### PR TITLE
Instance provision DSL

### PR DESCRIPTION
**Pull request by pun-ky**
usage of https://github.com/Cognifide/gradle-aem-plugin/pull/459

### CODE REVIEW

File: aem/gradle/environment.gradle.kts [hunk 1]
```diff
@@ -47,6 +47,26 @@ configure<AemExtension> {
             dependsOn(":develop")
         }
 
+        instanceSatisfy {
+            packages {
+                // https://github.com/Cognifide/gradle-aem-plugin#task-instancesatisfy
+            }
+        }
+
+        instanceProvision {
```
Comment (pun-ky) [inline aem/gradle/environment.gradle.kts]: @mierzwid what do you think? :)

Comment (mierzwid) [reply aem/gradle/environment.gradle.kts]: I like it very much. Well described steps, not a custom code.

File: .idea/runConfigurations/instance_provision.xml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+<component name="ProjectRunConfigurationManager">
```
Comment (mierzwid) [inline .idea/runConfigurations/instance_provision.xml]: Why you adding those intellij files?

Comment (pun-ky) [reply .idea/runConfigurations/instance_provision.xml]: image

for having such nice integration in IntelliJ

no longer need to write `sh gradlew` ... :)

**Pull request merged**